terraform.tfvars

```diff
@@ -0,0 +1,3 @@
+//region           = "us-east-1"  # See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
+//private_key_path = "/path/to/your/key"
+//public_key_path  = "/path/to/your/key.pub"
```

Comment:
Did you mean to include this file/you're referencing AWS docs?

Reply:
Ahh good catch.  Can you update that as  you are the alibaba guy. @shinyfoil

Reply:
Done. Closed.